Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -207,9 +207,10 @@ var ScrollView = React.createClass({
     scrollEnabled: PropTypes.bool,
     /**
      * This controls how often the scroll event will be fired while scrolling
-     * (in events per seconds). A higher number yields better accuracy for code
+     * (as a time interval in ms). A lower number yields better accuracy for code
      * that is tracking the scroll position, but can lead to scroll performance
      * problems due to the volume of information being send over the bridge.
+     * You should NOT set this value below 16ms as you will be wasting frames.
```

Comment:
I'm not sure if this is needed since the JS's run loop is synced to the screen refresh rate.